apps/src/templates/instructions/TeacherFeedback.jsx

```diff
@@ -268,7 +268,7 @@ class TeacherFeedback extends Component {
         : styles.timeTeacher;
 
     return (
-      <div style={tabVisible}>
+      <div id="unit-test-teacher-feedback-div" style={tabVisible}>
```

Comment:
I'd recommend removing this id for a couple reasons.

First, it's unnecessary; you should be able to do something like `wrapper.children().first()` or otherwise just navigate the element tree in the test to find this element.

Second, even if you _do_ have to add an externally-visible identifier to your component for your tests to work, it should always be a class, not an id. React components are inherently reusable, so you should expect that there's always a chance for multiple of them to show up on the same page.

Reply:
Yeah, I'd say it's a necessary evil for UI tests; but in this case it's being used by a _unit_ test, for which better alternatives exist.